source/_integrations/switchbot.markdown

```diff
@@ -47,6 +47,10 @@ Please note, device names configured in the SwitchBot app are not transferred in
 
 {% include integrations/config_flow.md %}
 
+SwitchBot devices are added to Home Assistant according to their domain, such as Curtains under the cover domain. For more information how to use these in automations, see the buliding block integration that domain, such as [cover](/integrations/cover/).
```

Comment:
This sentence feels odd and more like a base concept and not related to SwitchBot at all. IMHO, adding this isn't right, as it causes this to be added to each entity platform we have for each integration page.

Reply:
I'm curious what you think of this other PR #31789 I made to do address this idea in a more general/automated way. I just feel it's unclear for newer users if they end up on any specific integration page like Switchbot, there is no mention of how to automate with the entities it made.
Thanks for the feedback!